tests/test_data/software_plan.md

```diff
@@ -0,0 +1,588 @@
+---
```

Comment:
Once we have an integration test that runs against the real files, it would seem we should remove this file since it would be redundant, right?

Reply:
I think it is already redundant.